### ISSUE TITLE
3.9 Readme Instructions unclear

### ISSUE THREAD

**Issue by Joshua-Beha** [sclorg/s2i-python-container#580]
Building the sample app in OpenShift with the command specified by the readme (`oc new-app python:3.9~https://github.com/sclorg/django-ex.git`) fails with error `error: the image match "python-ubi/python:3.9" for source repository "https://github.com/sclorg/django-ex" does not appear to be a source-to-image builder.`. 
Curling the image registry (`curl -X GET -u user:passwd https://<registry-host>/v2/openshift/python/tags/list`) returns the following tags:
```
{
  "name": "openshift/python",
  "tags": [
    "3.8",
    "3.8-ubi8",
    "3.9-ubi9",
    "3.9-ubi8",
    "latest",
    "2.7",
    "2.7-ubi7",
    "2.7-ubi8",
    "3.6-ubi8",
    "3.8-ubi7"
  ]
}
```
Notably, this does *not* include the `3.9` tag.
The cluster has not had any changes to the image registry to the best of my knowledge. Could the README be updated to use a command that would work by default such as `oc new-app python:3.9-ubi8~https://github.com/sclorg/django-ex.git`? Alternatively, a further explanation could be provided as to why the `python:3.9` tag is not available by default.

**Comment by pkubatrh** [comment]
The default image streams are afaik tied to the release of OCP/Openshift. So you might have an older version that does not have it yet, or the latest version of Openshift did not integrate it yet.

The README expects you have the image stream available. If you do not, you can import ones maintained in this repository:  https://github.com/sclorg/s2i-python-container/tree/master/imagestreams

But I agree it makes sense to explicitly mention this in the README

**Comment by frenzymadness** [comment]
I'm sorry, but my knowledge of OpenShift is limited. Could please any of you open a PR here if there is anything to fix in the readme?

**Comment by pkubatrh** [comment]
Looking at this a second time (after getting some coffee) I see that the issue is not in the version of Openshift, but that the imagestream does not provide the `3.9` stream, but does provide `3.9-ubi8` and `3.9-ubi9` streams. I vaguely remember that this was done to actually force the users to differentiate between what platform they want to use.
So yes, we should update the README with the correct stream names, so that it works out of the box.

**Comment by Joshua-Beha** [comment]
@pkubatrh Forcing users to choose a platform would make sense. Any chance you would have a link to documentation explaining this? I can create the PR but I think having some documentation of the change on the OpenShift end would help to justify the change.

**Comment by Joshua-Beha** [comment]
@frenzymadness I have created the pull request, not sure what I need to do for reviews, etc.

**Comment by Joshua-Beha** [comment]
Resolved in PR #582
